src/pages/FaqPage/__tests__/hashScroll.test.tsx

```
@@ -0,0 +1,94 @@
+  scrollToMock = vi.fn();
+  return {
+    scroller: { scrollTo: scrollToMock },
+    Link: ({ children }: any) => <a>{children}</a>,
+  };
+});
+
+vi.mock("@/aaLogic/getPlotPrice", () => ({
+  getPlotPrice: () => ({ fee: 0 }),
+}));
+vi.mock("@/components/layout/page-layout", () => ({
+  PageLayout: ({ children }: any) => <div>{children}</div>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)